### PR TITLE
Fixes after the first install of PBM

### PR DESCRIPTION
**Pull request by boris-ilijic**
These are minor issues that I noticed during my first installation on local machine using docker. Please see commit message for additional explanation, if needed.

### CODE REVIEW

File: docs/details/cli.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ configuration-like step that should be done for practical ease though. (Without
 `PBM_MONGODB_URI`, the `--mongodb-uri` command line argument will need to
 be specified each time.)
 
-To learn how to set the `PBM_MONGODB_URI` environment variable, see [Set the MongoDB connection URI for `pbm` CLI](../install/initial-setup.md#set-mongodburi-pbm-cli). For more information about MongoDB URI connection strings, see [Authentication](authentication.md).
\ No newline at end of file
+To learn how to set the `PBM_MONGODB_URI` environment variable, see [Set the MongoDB connection URI for `pbm` CLI](../install/configure-authentication.html#set-the-mongodb-connection-uri-for-pbm-cli). For more information about MongoDB URI connection strings, see [Authentication](authentication.md).
```
Comment (nastena1606) [inline docs/details/cli.md:16]:
```suggestion
To learn how to set the `PBM_MONGODB_URI` environment variable, see [Set the MongoDB connection URI for `pbm` CLI](../install/configure-authentication.md#set-the-mongodb-connection-uri-for-pbm-cli). For more information about MongoDB URI connection strings, see [Authentication](authentication.md).
```

File: docs/install/configure-authentication.md [hunk 2]
```diff
@@ -129,7 +129,7 @@ Use the following command:
 export PBM_MONGODB_URI="mongodb://pbmuser:secretpwd@localhost:27017/?authSource=admin&replSetName=xxxx"
 ```
 
-For more information about what connection string to specify, refer to the [pbm connection string](../details/authentication.md#mongodb-connection-strings-a-reminder-or-primer) section.
+For more information about what connection string to specify, refer to the [pbm connection string](../details/authentication.html#mongodb-connection-strings) section.
```
Comment (nastena1606) [inline docs/install/configure-authentication.md:132]:
```suggestion
For more information about what connection string to specify, refer to the [pbm connection string](../details/authentication.md#mongodb-connection-strings) section.
```

**Review by nastena1606 — COMMENTED** [review]
Thanks for the PR!. Changed html to md in links since we're referring to source files which are then translated to respective format duting doc build

**Review by sandraromanchenko — APPROVED** [review]

**Comment by boris-ilijic** [comment]
@nastena1606 Good catch, wasn't aware of that. Should be fixed now.

**Review by nastena1606 — APPROVED** [review]
LGTM thanks!

**Pull request merged**